src/redux/middleware/sync.js

```diff
@@ -0,0 +1,58 @@
+import {
+  ACTIONS as LBRY_REDUX_ACTIONS,
+  makeSelectIsFollowingTag,
+  selectFollowedTags,
+} from 'lbry-redux';
+import Lbryio from 'lbryio';
+import * as ACTIONS from 'constants/action_types';
+import { selectSubscriptions } from 'redux/selectors/subscriptions';
+
+const persistShape = {
+  version: '0',
+  shared: {},
+};
+
+export function userStateSyncMiddleware() {
+  return ({ getState }) => next => action => {
+    if (
+      action.type === ACTIONS.CHANNEL_SUBSCRIBE ||
```

Comment:
you can write this as `[ACTIONS.CHANNEL_SUBSCRIBE, ACTIONS.CHANNEL_UNSUBSCRIBE, LBRY_REDUX_ACTIONS.TOGGLE_TAG_FOLLOW].includes(action.type)`

same for the one above

(it's not required or even strongly suggested to do the above, just sharing)